Makefile

```diff
@@ -0,0 +1,53 @@
+## Copyright 2020 Red Hat, Inc.
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+##     http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+
+-include Makefile.overrides
```

Comment:
This seems to be leftover from somewhere? File is not included in the PR

Reply:
`-include` means it's optional, so you can have a local includes file (e.g. something that always sets `CONTAINER_CLI=podman`).  No missing file, it's bring your own.